fixbackend/organizations/router.py

```diff
@@ -167,4 +167,26 @@ async def accept_invitation(
 
         return None
 
+    @router.get("/{organization_id}/external_id")
+    async def get_externa_id(
+        organization_id: UUID,
+        user_context: AuthenticatedUser,
+        organization_service: OrganizationServiceDependency,
+    ) -> ExternalId:
+        """Get an organization's external id."""
+        org = await organization_service.get_organization(organization_id)
+        if org is None:
+            raise HTTPException(status_code=404, detail="Organization not found")
+
+        if user_context.user.email not in [owner.user.email for owner in org.owners]:
+            raise HTTPException(
+                status_code=403, detail="You must be an owner of this organization to get an external ID"
+            )
+
+        return ExternalId(external_id=org.external_id)
+
+    @router.get("/{organization_id}/cf_url")
+    async def get_cf_url(organization_id: UUID, user_context: AuthenticatedUser) -> str:
+        return "https://example.com"
+
```

Comment:
Don't we need an endpoint to get account id, name, external_id and role_arn?

Reply:
This one will point to the cloud formation template, which is not there yet. You probably talking about the callback, let me add it here.